### PR TITLE
B-08223 : Story Generation - Story Submission and Response plus Cypress Tests

### PR DESCRIPTION
**Pull request by Jyaheza**

### CODE REVIEW

File: src/views/StoriesList.vue [hunk 1]
```diff
@@ -85,6 +68,72 @@ async function getStories() {
...
+}
+
+async function generateStory() {
+  loading.value = true;
+  try {
+    await StoryServices.addStory(user.value.id, {
+      storyParams: [
+        { key: 'setting', value: newStory.value.setting },
+        { key: 'country', value: newStory.value.country },
+        { key: 'lead character', value: newStory.value.leadCharacter },
```
Comment (markyd-XXI) [inline src/views/StoriesList.vue:100]: Looking at the dB and models, all we have for story attributes is characters, countries, languages and settings.

File: src/views/StoriesList.vue [hunk 2]
```diff
@@ -1,63 +1,46 @@
...
   text: "",
 });
-
 const newStory = ref({
-  setting: '',
-  leadCharacterName: '',
-  leadCharacterSpecies: '',
-  secondaryCharacter: '',
-  characterObjective: '',
+  setting: "",
```
Comment (markyd-XXI) [inline src/views/StoriesList.vue:23]: Setting, character, language, and country only probably.

File: src/components/StoryCardComponent.vue [hunk 3]
```diff
@@ -35,7 +35,8 @@ function navigateToStoryDetail() {
             {{ `${story.title}` }}
         </v-card-title>
         <v-card-text class="body-1">
-            {{ `${story.story.substring(0, 100)}... ` }}
+            <span v-if="showStory">{{ story.story }}</span>
```
Comment (markyd-XXI) [inline src/components/StoryCardComponent.vue:38]: Can we take this out?  I think the showStory was there initially to do an expand/collapse of the card. I'm not sure it's going to do anything. Clicking on the story takes you to a different view.

**Review by markyd-XXI — CHANGES_REQUESTED** [review]
Just a couple of minor things to change @Jyaheza.

**Review by markyd-XXI — APPROVED** [review]

File: src/views/StoriesList.vue [hunk 4]
```diff
@@ -1,63 +1,44 @@
...
-
-function closeSnackBar() {
-  snackbar.value.value = false;
-}
-
-function viewStory(storyId) {
-  router.push({ name: 'readStory', params: { id: storyId } });
-}
-
 async function getStories() {
```
Comment (n-grounds) [inline src/views/StoriesList.vue:42]: a get* method is generally expected to fetch and return data, while this seems to be fetching stories and then updating the page/view to show them, a better function name would be `loadStories`

**Comment by n-grounds** [comment]
forgot to squash :O

**Pull request merged**